extra/string-extra/Tests/Fixtures/plural-invalid-language.test

```diff
@@ -0,0 +1,10 @@
+--TEST--
+"plural" filter
+--TEMPLATE--
+{{ 'partition'|plural('it') }}
```

Comment:
If that can motivate someone to implement pluralization in Italian, just to break this test 😂